packages/indy-vdr/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@aries-framework/indy-vdr",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.2.5",
+  "files": [
+    "build"
+  ],
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://github.com/hyperledger/aries-framework-javascript/tree/main/packages/indy-vdr",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperledger/aries-framework-javascript",
+    "directory": "packages/indy-vdr"
+  },
+  "scripts": {
+    "build": "yarn run clean && yarn run compile",
+    "clean": "rimraf -rf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "yarn run build",
+    "test": "jest"
+  },
+  "dependencies": {
+    "class-transformer": "0.5.1",
+    "indy-vdr-test-shared": "^0.1.3",
+    "class-validator": "0.13.1",
+    "rxjs": "^7.2.0"
+  },
+  "peerDependencies": {
+    "@aries-framework/core": "0.2.5",
+    "indy-vdr-test-nodejs": "^0.1.3"
+  },
+  "devDependencies": {
+    "@aries-framework/node": "0.2.5",
+    "indy-vdr-test-nodejs": "^0.1.3",
+    "reflect-metadata": "^0.1.13",
+    "rimraf": "~3.0.2",
+    "typescript": "~4.3.0"
```

Comment:
You will also need to update this line to "typescript": "~4.9.4" to align with other packages. Also for the rimraf (you'll need to remove the -rf parameter in the clean test script)